test/rules/no-shadowed-variable/test.ts.lint

```diff
@@ -118,13 +139,16 @@ interface FalsePositive4<T, TResult> {
 let p = 1;
 function testParameterDestructuring(
     { pos: p }: FalsePositive3, // failure
+           ~                               [shadowed variable: 'p']
     { pos: q, specular: r }: FalsePositive3,
     { pos }: FalsePositive3
 ) {
     p = 2;
     var q = 1; // failure
+        ~                 [shadowed variable: 'q']
 
     function someInnerFunc() {
         let pos = 3; // failure
+            ~~~                 [shadowed variable: 'pos']
     }
-}
+}
```

Comment:
can you try to fix your editor settings to keep the trailing newlines? we need to add an `.editorconfig` #940